span.go

```diff
@@ -49,22 +53,35 @@ type Span interface {
 	// may ignore the tag, but shall not panic.
 	SetTag(key string, value interface{}) Span
 
-	// LogEvent() is equivalent to
+	// LogFields is an efficient and type-checked way to record key:value
+	// logging data about a Span, though the programming interface is a little
+	// more verbose than LogKV(). Here's an example:
 	//
-	//   Log(LogData{Event: event})
+	//    span.LogFields(
+	//        log.String("request_path", request.Path()),
```

Comment:
Same concern about tags vs logging as per above. Is structured logging actually supposed to replace tags? Because I didn't see that in the proposal.